supabase/migrations/20231017171314_my_table.sql

```diff
@@ -0,0 +1,54 @@
+drop policy "Enable read access for authenticated users." on "public"."pallet";
+
+drop policy "Enable insert for authenticated users only" on "public"."project";
+
+drop policy "Enable read access for authenticated users." on "public"."shipment";
+
+alter table "public"."pallet" disable row level security;
+
+alter table "public"."pallet_scan" disable row level security;
+
+alter table "public"."project" disable row level security;
+
+alter table "public"."shipment" add column "bookkeeping_flexport" boolean;
+
+alter table "public"."shipment" add column "bookkeeping_gik" boolean;
+
+alter table "public"."shipment" add column "commercial_invoice_link" text;
+
+alter table "public"."shipment" add column "consignee" text;
+
+alter table "public"."shipment" add column "destination" text not null;
+
+alter table "public"."shipment" add column "doner" text;
+
+alter table "public"."shipment" add column "drive_link" text;
+
+alter table "public"."shipment" add column "estimated_delivery_date" timestamp with time zone;
+
+alter table "public"."shipment" add column "flexport_tracking_id" text;
+
+alter table "public"."shipment" add column "freight_tracking_id" text;
+
+alter table "public"."shipment" add column "gik_value" text;
+
+alter table "public"."shipment" add column "impact_reporting" boolean;
+
+alter table "public"."shipment" add column "invoice_link" text;
+
+alter table "public"."shipment" add column "logistics_expense_donation" text;
+
+alter table "public"."shipment" add column "main_shipment_type" text;
```

Comment:
We probably want to create a `shipment_types` table and reference from there, rather than a free text field.